app/main.py

```diff
@@ -1 +1,21 @@
-# write your code here
+from app.cafe import Cafe
+from app.errors import (NotWearingMaskError,
+                        NotVaccinatedError,
+                        OutdatedVaccineError)
+
+
+def go_to_cafe(friends, cafe) -> str:
```

Comment:
Please, use type hints for functions you declare. Ideally specifying type of objects inside the list
```suggestion
def go_to_cafe(friends: list, cafe: Cafe) -> str:
```